setup-homebrew/main.sh

```diff
@@ -31,8 +31,15 @@ function git_retry {
 
 # Check brew's existence
 if ! which brew &>/dev/null; then
-    echo "Could not find 'brew' command in PATH."
-    exit 1
+    echo "Could not find 'brew' command in PATH. Sourcing from default locations."
+    [[ -f "/usr/local/bin/brew" ]] && eval "$(/usr/local/bin/brew shellenv)"
+    [[ -f "/opt/Homebrew/bin/brew" ]] && eval "$(/opt/Homebrew/bin/brew shellenv)"
+    [[ -f "/home/linuxbrew/.linuxbrew/bin/brew" ]] && eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
```

Comment:
```suggestion
    if [[ -f "/usr/local/bin/brew" ]]; then 
        eval "$(/usr/local/bin/brew shellenv)"
    elif [[ -f "/opt/Homebrew/bin/brew" ]]; then 
        eval "$(/opt/Homebrew/bin/brew shellenv)"
    elif [[ -f "/home/linuxbrew/.linuxbrew/bin/brew" ]]; then
        eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
    fi
```

Reply:
`s/Homebrew/homebrew`. It'll work since the file system is (typically) case-insensitive, but I think this usually confuses `brew`.

Reply:
> I think you want to write to $GITHUB_PATH or $GITHUB_ENV too or else the environment changes don't make it past the action invocation.

That's line 63.

Reply:
I do mind a little. If the environment changes made my `brew shellenv` are only for the purposes of running this script then we should just do
```sh
export PATH="/home/linuxbrew/.linuxbrew/bin:/opt/homebrew/bin:/usr/local/bin:$PATH"
```
and let the rest of the script take care of setting `PATH` properly.

Reply:
Sounds good to me